File: applied/02_i_wanna_be_your_lover/answer/3.c
#include <stdio.h>
int main(void){
    int divider, quotient, remainder;
    scanf("%d", &divider);
    scanf("%d", &quotient);
    scanf("%d", &remainder);

    // dividee / divider = quotient
    // dividee           = quotient * divider
    // dividee           = quotient * divider + remainder
    printf("%d\n", divider * quotient + remainder);

    return 0;
}

File: applied/02_i_wanna_be_your_lover/answer/3.java
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner scan = new Scanner(System.in);
        
        int divider = scan.nextInt();
        int quotient = scan.nextInt();
        int remainder = scan.nextInt();
        
        // dividee / divider = quotient
        // dividee           = quotient * divider
        // dividee           = quotient * divider + remainder
        System.out.printf("%d\n", divider * quotient + remainder);

        scan.close();
    }
}
